.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [2.17.7, dev]
+        sdk: [stable, dev]
```

Comment:
Call this 3.0.0 - stable will change!

Reply:
Fair point, done! Can you tell why the workflow isn't running though? Not sure what I'm missing.

Reply:
zero clue 🤷